frontend/e2e/validation.spec.ts

```diff
@@ -0,0 +1,60 @@
+import { test, expect } from "@playwright/test"
+
+test.describe("validation errors", () => {
```

Comment:
Todo: this tests validation on a specific page, right? So it should be in that page's test file or at least be named more precisely